api/v1/_submissions/PKPBackendSubmissionsHandler.php

```diff
@@ -108,18 +134,39 @@ public function getMany($slimRequest, $response, $args)
             return $response->withStatus(404)->withJsonError('api.404.resourceNotFound');
         }
 
-        $collector = $this->getSubmissionCollector($slimRequest->getQueryParams());
+        $queryParams = $slimRequest->getQueryParams();
 
-        // Anyone not a manager or site admin can only access their assigned
-        // submissions
-        $userRoles = $this->getAuthorizedContextObject(Application::ASSOC_TYPE_USER_ROLES);
-        $canAccessUnassignedSubmission = !empty(array_intersect([Role::ROLE_ID_SITE_ADMIN, Role::ROLE_ID_MANAGER], $userRoles));
-        Hook::call('API::submissions::params', [$collector, $slimRequest]);
-        if (!$canAccessUnassignedSubmission) {
-            if (!is_array($collector->assignedTo)) {
-                $collector->assignedTo([$currentUser->getId()]);
-            } elseif ($collector->assignedTo != [$currentUser->getId()]) {
-                return $response->withStatus(403)->withJsonError('api.submissions.403.requestedOthersUnpublishedSubmissions');
+        $collector = $this->getSubmissionCollector($queryParams);
+
+        foreach ($queryParams as $param => $val) {
+            switch ($param) {
+                case 'assignedTo':
+                    $val = array_map('intval', $this->paramToArray($val));
+                    if ($val == [\PKP\submission\Collector::UNASSIGNED]) {
+                        $val = array_shift($val);
+                    }
+                    $collector->assignedTo($val);
+                    break;
+
+                case 'isIncomplete':
+                    $collector->filterByIncomplete(true);
+                    break;
+            }
+        }
+
+        // In new submission lists this endpoint is dedicated to retrieve all submissions only by admins and managers
+        if (!Config::getVar('features', 'enable_new_submission_listing')) {
```

Comment:
To decrease the chances of parts of this getting left in the `main` codebase, please file an issue against the `3.5 Internal` milestone for cleanup, and add FIXME notes to the parts of the code that'll require cleanup before release:
```
// FIXME: Clean up before release pkp/pkp-lib#1234
```
That way we can grep for cases like this and be sure we get them all.